packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -1021,61 +1032,133 @@ export default class Meeting extends StatelessWebexPlugin {
     this.locusInfo.on(EVENTS.LOCUS_INFO_UPDATE_MEDIA_SHARES, (payload) => {
       const {contentId, disposition} = payload.current;
 
-      // Check if screenshare is being taken over by a remote
-      // If so, terminate the share and allow the remote to be on the floor
-      // Terminating the share allows the user to screenshare again without any issues
       if (
-        this.isSharing &&
-        this.selfId !== contentId &&
-        disposition !== FLOOR_ACTION.RELEASED
+        payload.previous &&
+        contentId === payload.previous.contentId &&
+        disposition === payload.previous.disposition
```

Comment:
```suggestion
        contentId === payload.previous?.contentId &&
        disposition === payload.previous?.disposition
```

Reply:
hmm.. ok, this won't be exactly the same as I intended when writing this code, because if payload.previous is undefined and we get contentId and disposition as undefined too, then your proposed condition will evaluate to true, while the current one evaluates to false, but I guess it's OK as we never get disposition as undefined from locus (we get it as null if no one ever shared in the meeting) and also the later code in this function checks disposition against either released or granted in all the conditions, so it should be fine.